lint-staged.config.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const path = require('path');
+const fs = require('fs');
+const findUp = require('find-up');
+
+const includes = ['packages', '.github'];
+
+const root = path.resolve(__dirname);
+
+function extractPackageName(pkgJsonPath) {
+  return JSON.parse(fs.readFileSync(pkgJsonPath).toString()).name;
+}
+
+function getLintCommand(files) {
+  const affectedFolders = new Set();
+
+  for (const file of files) {
+    const r = findUp.sync('package.json', { cwd: file });
+    const relPath = path.relative(root, r);
+
+    if (includes.some((incl) => relPath.startsWith(incl))) {
+      affectedFolders.add(r);
+    }
+  }
+
+  const affectedPackages = [...affectedFolders].map(extractPackageName);
+
+  if (affectedPackages.length === 0) {
+    return null;
+  }
+  return `nx run-many -t lint -p ${affectedPackages.join()} --parallel 5`;
```

Comment:
why did you choose 5 to run in parallel? 🤔

Reply:
I would force it in the 2 in github actions since you only have 2 cores  using more then parallel  2 would only make it slower 

assuming that every parallel is 1 working

I would for normal pcs put it on 50% otherwise to use 50% of the cores on a user running the command

Reply:
No sure 🤔  I think a tried different values and it was the best for my computer. I'll remove to use the default to 3 @Boegie19 this is only something that runs locally here

Reply:
3 feels a bit arbitrary too. What do you think about setting this to the number of CPUs available on the host system? `os.cpus().length`

Reply:
Alright I see you set it back to default. Nevermind then.